ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/tree/nodes/pairs/values/TomlBasicString.kt

```diff
@@ -43,19 +46,25 @@ public class TomlBasicString internal constructor(
     public companion object {
         private fun String.verifyAndTrimQuotes(lineNo: Int): Any =
                 when {
+                    // ====== multiline string (""") =======
```

Comment:
Yeah, I see: you decided not to make differences on the higher level. 
That's actually okay, may I should have done the same with LiteralStrings